_posts/2019-02-11-bootstrap-4-3-0.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Bootstrap 4.3.0
+video: htgr3pvBr-I
+---
+
+Bootstrap v4.3 has landed with over 120 combined closed issues and merged pull requests. This release brings improvements to our utilities, some prep work for moving on to v5's development, and the standard bug fixes and documentation updates.
+
+During our last release, we shared a small preview of where we're taking the project next. That's getting clearer in the coming weeks as our attention turns towards embracing Hugo for ultra fast docs development, removing jQuery in favor of regular JavaScript, and addressing our growing code base.
+
+Keep reading for v4.3 highlights, and see you soon with more details on v5!
+
+## Highlights
+
+We've added some new utilities and deprecated some unused code. Here are the key changes in v4.3, broken down by new, improved, fixed, and deprecated.
+
+- **New:** Added `.stretched-link` utility to make any anchor the size of it's nearest `position: relative` parent, perfect for entirely clickable cards!
+- **New:** Added `.text-break` utility for applying `word-break: break-word`
+- **New:** Added `.rounded-sm` and `.rounded-lg` for small and large `border-radius`.
+- **New:** Added `.modal-dialog-scrollable` modifier class for scrolling content _within_ a modal.
+- **New:** Added responsive `.list-group-horizontal` modifier classes for displaying list groups as a horizontal row.
+- **Improved:** Reduced our compiled CSS by using `null` for variables that by default inherit their values from other elements (e.g., `$headings-color` was `inherit` and is now `null` until you modifier it in your custom CSS).
+- **Improved:** Simplified popover arrow CSS by generating arrows on the `.popover` instead of an additional element, `.popover-arrow`.
+- **Improved:** Badge focus styles now match their `background-color` like our buttons.
+- **Fixed:** Reverted v4.2.1's change to the breakpoint and grid container Sass maps that blocked folks from upgrading when modifying those default variables.
+- **Fixed:** Restored `white-space: nowrap` to `.dropdown-toggle` (before v4.2.1 it was on all `.btn`s) so carets don't wrap to new lines.
+- **Deprecated:** `img-retina`, `invisible`, `float`, and `size` mixins are now deprecated and will be removed in v5.
+
+Checkout the full [v4.3.0 ship list](https://github.com/twbs/bootstrap/issues/27893) and [GitHub project](https://github.com/twbs/bootstrap/projects/16) for the full details.
+
+[Head to to the v4.3.x docs](https://getbootstrap.com/docs/4.3/) to see the latest in action. The full release has been published to npm and will soon appear on the Bootstrap CDN and Rubygems.
+
+## Introducing responsive font sizes
+
+![Responsive font-sizes](https://camo.githubusercontent.com/eb6458d9bb840c5b5764a33ac626ca084f12e460/68747470733a2f2f692e696d6775722e636f6d2f50326978614b482e706e67)
```

Comment:
Can you add the image to the repo please? That way we can track it, optimize etc